server/api/rounds.py

```diff
@@ -515,6 +515,7 @@ def draw_sample_for_contest(contest: Contest, sample_size: int) -> List[BallotDr
     samples = [
         draw_sample_for_contest(contest, sample_sizes[contest.id])
         for contest in contests
+        if contest.is_targeted
```

Comment:
this filter is already applied to the input for this function. see L465

Reply:
Will fix.